apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Trace/TraceTest.cs

```diff
@@ -70,6 +72,29 @@ public async Task Trace()
                 TraceEntryVerifiers.AssertSpanLabelsContains(
                     trace.Spans.First(s => s.Name == uri), TraceEntryData.HttpGetSuccessLabels);
 
+                Assert.False(response.Headers.Contains(TraceHeaderContext.TraceHeader));
+            }
+        }  
+        
+        [Fact]
+        public async Task Trace_dummy_name_provider()
```

Comment:
nit: This name should be Trace_DummyNameProvider to follow the same naming convetion we use elsewhere.

Reply:
👍 - force of naming habit. Last commit amended.